src/euphorie/client/browser/login.py

```diff
@@ -285,6 +285,7 @@ def _tryRegistration(self):
             account.loginname = loginname
             account.password = reply.get("password1")
             account.account_type = config.CONVERTED_ACCOUNT
+            account.created = datetime.datetime.now()
```

Comment:
What will be the created value for existing users?

Reply:
Null

Reply:
Unless @pilz has a differen opinion?